syncano_cli/parse_to_syncano/commands.py

```diff
@@ -1,52 +1,66 @@
 # -*- coding: utf-8 -*-
-import six
+import click
 from syncano_cli import LOG
-from syncano_cli.commands_base import CommandContainer, argument
-from syncano_cli.parse_to_syncano.config import config
+from syncano_cli.config import ACCOUNT_CONFIG_PATH
+from syncano_cli.parse_to_syncano.config import read_config
 from syncano_cli.parse_to_syncano.migrations.transfer import SyncanoTransfer
 from syncano_cli.parse_to_syncano.moses import check_configuration, force_configuration_overwrite, print_configuration
 
-COMMAND_NAMESPACE = 'import'
 
+@click.group()
+@click.pass_context
+def top_transfer(context):
+    pass
 
-class Parse(six.with_metaclass(CommandContainer)):
-    namespace = COMMAND_NAMESPACE
 
-    @classmethod
-    def run(self, context):
-        """
+@top_transfer.group()
+@click.pass_context
+@click.option('--config', help=u'Account configuration file.')
+def transfer(context, config):
```

Comment:
This is exactly what I'm searched for! Thanks. Previous one was import but this is the same as for class -> the reserved word.